api/log/zap.go

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package log
+
+import (
+	"os"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/apisix/manager-api/conf"
+)
+
+var logger *zap.SugaredLogger
+
+func init() {
+	writeSyncer := fileWriter()
+	encoder := getEncoder()
+	logLevel := getLogLevel()
+	core := zapcore.NewCore(encoder, writeSyncer, logLevel)
+
+	zapLogger := zap.New(core, zap.AddCaller())
+
+	logger = zapLogger.Sugar()
+}
+
+func getLogLevel() zapcore.LevelEnabler {
+	level := zapcore.WarnLevel
+	switch conf.ErrorLogLevel {
+	case "debug":
+		level = zapcore.DebugLevel
+	case "info":
+		level = zapcore.InfoLevel
+	case "error":
```

Comment:
`notice`, `warn`, `crit`

can we add more log level?

Reply:
log levels commonly used in Go libs:
debug info warn error panic fatal

what do you think ?

Reply:
do we have supported all of them? `debug info warn error panic fatal`

if yes, I think it is fine. please confirm

Reply:
ok, I will at them all.

Reply:
as we talked right now, we need to support them in this PR.